modules/otel-collector-components.adoc

```diff
@@ -199,6 +199,138 @@ The OpenCensus receiver provides backwards compatibility with the OpenCensus pro
 Wildcards with `+*+` are accepted under the `cors_allowed_origins`.
 To match any origin, enter only `+*+`.
 
+[id="k8scluster-receiver_{context}"]
+=== Kubernetes Cluster Receiver
+
+The Kubernetes Cluster Receiver gathers cluster metrics and entity events from the Kubernetes API server. It uses the Kubernetes API to receive information about updates. Authentication for this receiver is only supported through service accounts.
+
+.OpenTelemetry Collector custom resource with the enabled Kubernetes Cluster Receiver
+[source,yaml]
+----
+  receivers:
+    k8s_cluster:
+      distribution: openshift
+      collection_interval: 10s
+  exporters:
+    debug:
+  service:
+    pipelines:
+      metrics:
+        receivers: [k8s_cluster]
+        exporters: [debug]
+      logs/entity_events:
+        receivers: [k8s_cluster]
+        exporters: [debug]
+----
+
+This receiver requires a set up ServiceAccount, RBAC rules for the ClusterRole, and the ClusterRoleBinding that binds the RBAC with the ServiceAccount.
```

Comment:
ISG says "setup" when an adjective. Altho maybe "configured" would feel a little more natural here? 

Also marking up the [objects](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#object-references):
```suggestion
This receiver requires a configured `ServiceAccount` object, RBAC rules for the `ClusterRole` object, and the `ClusterRoleBinding` object that binds the RBAC with the `ServiceAccount`.
```
[Alternatively](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#general-references):
```suggestion
This receiver requires a configured service account, RBAC rules for the cluster role, and the cluster role binding that binds the RBAC with the service account.
```

Reply:
:slightly_smiling_face: I used `set up` as a participle.